tutorial/practicals/nemo/2_nemo_profiling/README.md

```diff
@@ -49,11 +49,11 @@ this session.)
        $ make allclean
        $ make tra_adv.exe
 
```

Comment:
The link at line 25 is not rendered properly by github, are "]\n(" not supported? Could you check if this is an issue somewhere else?